### DISCUSSION TITLE
Vertical alignment

### DISCUSSION THREAD

**Discussion by harrysw1729** [mcanouil/quarto-iconify#30] [Q&A] [answered]
Currently, `inline=true` is automatically added. According to [Iconify's docs](https://iconify.design/docs/iconify-icon/inline.html), this is equivalent to setting the style attribute with `vertical-align:-0.125em`. Is there any way to manually adjust the `vertical-align` style attribute?

Thanks a lot for the help!

**Comment by mcanouil** [comment, marked as answer]
You could overwrite this for now using span.
Something like `[shortcode]{style="whatever"}`.

**Reply by harrysw1729** [reply]
Thanks a lot! That works like a charm! The suggested workflow makes it very convenient to streamline the customization through css.

**Reply by mcanouil** [reply]
I might add "inline" as a direct option but span is the way to have all the CSS control you need or might want, for example, to add colour.

**Reply by mcanouil** [reply]
There is now a new boolean attribute in the shortcode: `{{< iconify fluent-emoji:exploding-head inline=false >}}`

See https://github.com/mcanouil/quarto-iconify/releases/tag/2.1.2
